dictionaries/software-terms/src/cybersecurity-terms.txt

```diff
@@ -1,4 +1,4 @@
-# Terms releated to Cyber Security
```

Comment:
It's funny to see some typo in your own repository

If only you had an amazing tool such as cspell to detect such errors 😅😬😂

Reply:
Exactly 🤦‍♂️. Checking is generally turned off for the src directories because they contain lots of things that should not be checked. But errors creep in.

Reply:
That's what errors does. They are so annoying.

I assumed errors checking was disabled, hits normal with such large sets of data, some are prefix/suffix, it would lead to validate words with themselves...

Maybe you could find a way to check everything following the last `#` character